assets/training/finetune_acft_multimodal/components/pipeline_components/multimodal_classification/spec.yaml

```diff
@@ -387,7 +387,7 @@ jobs:
 
   multimodal_classification_finetune:
     type: command
-    component: azureml:multimodal_classification_finetune:0.0.2
+    component: azureml:multimodal_classification_finetune@latest #:0.0.2
```

Comment:
Would "latest" work? I thought we need to specify the exact version

Reply:
Thanks for catching that. Fixed.